docs/site/MONOREPO.md

```diff
@@ -5,32 +5,36 @@ The [loopback-next](https://github.com/strongloop/loopback-next) repository uses
+| [repository-json-schema](https://github.com/strongloop/loopback-next/tree/master/packages/repository-json-schema)   | @loopback/repository-json-schema | Convert a TypeScript class/model to a JSON Schema                                                                                                                                                                                                                                                |
+| [repository](https://github.com/strongloop/loopback-next/tree/master/packages/repository)                           | @loopback/repository             | Define and implement a common set of interfaces for interacting with databases                                                                                                                                                                                                                   |
+| [rest](https://github.com/strongloop/loopback-next/tree/master/packages/rest)                                       | @loopback/rest                   | Expose controllers as REST endpoints and route REST API requests to controller methods                                                                                                                                                                                                           |
+| [service-proxy](https://github.com/strongloop/loopback-next/tree/master/packages/service-proxy)                     | @loopback/service-proxy          | A common set of interfaces for interacting with service oriented backends such as REST APIs, SOAP Web Services, and gRPC microservices                                                                                                                                                           |
+| [testlab](https://github.com/strongloop/loopback-next/tree/master/packages/testlab)                                 | @loopback/testlab                | A collection of test utilities we use to write LoopBack tests                                                                                                                                                                                                                                    |
+| [tslint-config](https://github.com/strongloop/loopback-next/tree/master/packages/tslint-config)                     | @loopback/tslint-config          | Shared TSLint config to enforce a consistent code style for LoopBack development                                                                                                                                                                                                                 |
```

Comment:
IMO, we should create a script to update this file as part of the build. See `loopback-next/bin/` for some similar ones.

Reply:
I'll look into this and make a separate PR in the future.